source/quickstart.txt

```diff
@@ -219,6 +220,34 @@ The API documentation provides details on using the following endpoints:
    * - :ref:`c2c-api-reverse`
      - Reverses the direction of a committed sync operation.
 
+One-Time Sync
+-------------
+
+For a one-time sync, after initializing data synchronization, use the
+:ref:`progress <c2c-api-progress>` command to see the status of the
+synchronization process: 
+
+.. literalinclude:: /includes/api/requests/progress.sh
+   :language: shell
+
+The following requirements must be met for a one-time sync: 
+
+  - ``state: "RUNNING"``
+  - ``canCommit: true``
+  - ``lagTimeSeconds`` is near ``0`` (*Recommended, but not required*)
+
+Then, use the :ref:`commit <c2c-api-commit>` command to commit the
+synchronization operation to the destination cluster and stop continuous
+replication:
+
+.. literalinclude:: /includes/api/requests/commit.sh
+   :language: shell
+
+If the :ref:`commit <c2c-api-commit>` request is sucessful,
```

Comment:
[nit] typo + [minimize repeated links](https://www.mongodb.com/docs/meta/style-guide/style/links-and-cross-references/#minimize-repeated-links)
```suggestion
If the ``commit`` request is successful,
```